tangy-input-groups.js

```diff
@@ -103,6 +103,9 @@ export class TangyInputGroups extends PolymerElement {
       inputGroupEl.addEventListener('input-group-remove', event => this.removeGroup(event.target.name))
       inputGroupEl.querySelectorAll('[name]').forEach(input => input.setAttribute('name', `${key}.${input.getAttribute('name')}`))
       this.appendChild(inputGroupEl)
+      this.addEventListener('click', (e) => {
+        this.dispatchEvent(new Event('change', { bubbles: true }))
+      })
```

Comment:
@evansdianga I like where you are going with this approach and see how it would solve for most inputs that involve a click event. I'm afraid some inputs might not involve a click event, especially if you are filling out the form using keyboard commands. How about the following which bubbles up any change events?

```javascript
inputGroupEl.querySelectorAll('[name]').forEach(input => {
  input.addEventListener('change', (e) => {
    this.dispatchEvent(new Event('change', { bubbles: true }))
  })
})
```

Reply:
For sure. Lemme work on it and make the change